pkgs/development/web/rover/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, perl
+, openssl
+, pkg-config
+, darwin
+, stdenv
+}:
+
+let
+  # The build fetches this dynamically from https://graphql.api.apollographql.com/api/schema
+  apolloSchema = builtins.readFile ./schema.graphql;
+  # The build fetches this from the headers of https://graphql.api.apollographql.com/api/schema
+  etagId = builtins.readFile ./etag.id;
```

Comment:
Why not use fetchers instead of downloading those files into nixpkgs.

Reply:
The problem is that the upstream file changes dynamically, I could throw it on ipfs or something if that's better!